CondCore/PPSTimingCalibrationPlugins/interface/PPSTimingCalibrationPayloadInspectorHelper.h

```diff
@@ -0,0 +1,141 @@
+#ifndef CONDCORE_PPSTIMINGCALIBRATIONPLUGINS_PPSTIMINGCALIBRATIONPAYLOADINSPECTORHELPER_H
+#define CONDCORE_PPSTIMINGCALIBRATIONPLUGINS_PPSTIMINGCALIBRATIONPAYLOADINSPECTORHELPER_H
+
+// User includes
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "CondCore/Utilities/interface/PayloadInspectorModule.h"
+#include "CondCore/Utilities/interface/PayloadInspector.h"
+#include "CondCore/CondDB/interface/Time.h"
+#include "CondFormats/PPSObjects/interface/PPSTimingCalibration.h"
+
+// system includes
+#include <fmt/printf.h>
+#include <memory>
+#include <sstream>
+
+// ROOT includes
+#include "TCanvas.h"
+#include "TStyle.h"
+#include "TH2F.h"
+#include "TLatex.h"
+#include "TGraph.h"
+
+namespace PPSTimingCalibrationPI {
+
+  enum parameter { parameter0 = 0, parameter1 = 1, parameter2 = 2, parameter3 = 3 };
+
+  enum conditions_db { db0 = 0, db1 = 1 };
+
+  enum conditions_plane { plane0 = 0, plane1 = 1, plane2 = 2, plane3 = 3 };
+
+  enum conditions_channel {
+    channel0 = 0,
+    channel1 = 1,
+    channel2 = 2,
+    channel3 = 3,
+    channel4 = 4,
+    channel5 = 5,
+    channel6 = 6,
+    channel7 = 7,
+    channel8 = 8,
+    channel9 = 9,
+    channel10 = 10,
+    channel11 = 11
+
+  };
+
+  inline std::string getStringFromParamEnum(const parameter& parameter) {
+    switch (parameter) {
+      case 0:
+        return "parameter no.0";
+      case 1:
+        return "parameter no.1";
+      case 2:
+        return "parameter no.2";
+      case 3:
+        return "parameter no.3";
+
+      default:
+        return "not here";
+    }
+  }
+
+  /************************************************
+    History plots
+  *************************************************/
+  template <conditions_db db, conditions_plane plane, conditions_channel channel, parameter param, class PayloadType>
+  class parametersPerRun : public cond::payloadInspector::HistoryPlot<PayloadType, float> {
+  public:
+    parametersPerRun()
+        : cond::payloadInspector::HistoryPlot<PayloadType, float>(getStringFromParamEnum(param) + "vs. Runs",
+                                                                  getStringFromParamEnum(param)) {}
+
+    float getFromPayload(PayloadType& payload) override { return payload.parameters(db, 1, plane, channel)[param]; }
+  };
+
+  /************************************************
+    X-Y correlation plots
+  *************************************************/
+  template <conditions_db db,
+            conditions_plane plane,
+            conditions_channel channel,
+            parameter param1,
+            parameter param2,
+            class PayloadType>
+  class pppCorrelation : public cond::payloadInspector::ScatterPlot<PayloadType, double, double> {
```

Comment:
Class names should start with uppercase character, as specified in https://cms-sw.github.io/cms_coding_rules.html